src/components/Main/NavCircle/NavCircle.tsx

```diff
@@ -1,19 +1,173 @@
 import React from 'react';
+import styled from 'styled-components';
 
-// TODO: Add circle background
-// TODO: Add three links
-// TODO: Add social media links
+// TODO: Add circle background (done)
+// TODO: Add three links 
+// TODO: Add social media links (done)
 // TODO: (if needed in a future PR) depending on scroll position, change the color of circle border
-// TODO: Make absolute on the page so it follows the screen
+// TODO: Make absolute on the page so it follows the screen (done)
+
+const StyledLinkOne = styled.a`
+  position: absolute;
+    top: 25%;
+    left: 48%;
+    margin-right: -50%;
+    margin-left: 0.5%;
+    transform: translate(-50%, -50%)
+    
+
+`
+const StyledLinkTwo = styled.a`
+position: absolute;
+    top: 40%;
+    left: 53.5%;
+    
+    margin-right: -60%;
+    margin-left: 1%;
+    transform: translate(-50%, -50%)
+`
+
+const StyledLinkThree = styled.a`
+position: absolute;
+    top: 55%;
+    left: 50%;
+    
+    margin-right: -50%;
+    transform: translate(-50%, -50%)
+`
+const StyledLinkFour = styled.div`
+position: absolute;
+    top: 70%;
+    left: 42%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%)
+`
+
+const StyledLinkFive = styled.div`
+position: absolute;
+    top: 70%;
+    left: 57%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%)
+`
+
+const Insta = () => {
+  return(
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 48 48"
+      width={36}
+      height={36}
+    >
+    <a href="https://www.instagram.com/gwcuva/?hl=en" target="_blank" rel="noreferrer"><img src="#" alt="Instagram" />
+      <path
+        d="M16 6C10.477 6 6 10.477 6 16v16c0 5.523 4.477 10 10 10h16c5.523 0 10-4.477 10-10V16c0-5.523-4.477-10-10-10zm20 4a1.999 1.999 0 110 4 1.999 1.999 0 110-4zm-12 4c5.523 0 10 4.477 10 10s-4.477 10-10 10-10-4.477-10-10 4.477-10 10-10zm0 4c-3.313 0-6 2.688-6 6 0 3.313 2.688 6 6 6 3.313 0 6-2.688 6-6 0-3.313-2.688-6-6-6zm0 0"
+        fill="#003046"
+      />
+      </a>
+    </svg>
+  )
+}
+
+const Discord = () => {
```

Comment:
instead of making these components, can you save the svg and import the same way that Navbar.tsx does?